src/solver/lbfgs.jl

```diff
@@ -1,6 +1,7 @@
 export lbfgs
 
 function lbfgs(nlp :: AbstractNLPModel;
+               logger :: AbstractLogger=NullLogger(),
```

Comment:
This is great but how are we passing this keyword argument to the solver? `solve_problems` already takes a keyword argument named `logger`. I guess it also needs to take one named, e.g., `solver_logger`, and pass that to the solver: `solver(nlp, logger=solver_logger)`?!

Reply:
In this case I was thinking that we probably wouldn't want the solver log, because it would print every iteration of every problem.

Reply:
I agree. It would still be great to get the output if we want to (say, if we just solve 2 or 3 problems). So `solver_logger` could be `NullLogger()` by default. What do you think?

Reply:
:+1: